lib/gman_client.rb

```diff
@@ -5,12 +5,16 @@
 
 # GmanClient
 module GmanClient
-  @url = ENV['WRAP_URL']
-  @token_url = ENV['TOKEN_URL']
-  @client_id = ENV['CLIENT_ID']
-  @client_secret = ENV['CLIENT_SECRET']
-  class << self
+  # GC
+  class GC
```

Comment:
I was mistaken. Changing now. I believe my first version used a gem with Client reserved and that added to my confusion. Thanks!